Makefile

```diff
@@ -507,24 +502,21 @@ KBUILD_CFLAGS   := -Wall -Wundef -Werror=strict-prototypes -Wno-trigraphs \
 		   -Werror=return-type -Wno-format-security \
 		   -std=gnu89
 KBUILD_CPPFLAGS := -D__KERNEL__
-KBUILD_RUSTCFLAGS :=
-# TODO: a simple way to update `Cargo.lock` when we add a new driver
-KBUILD_CARGOFLAGS := $(CARGO_VERBOSE) --locked \
-		    -Z build-std=core,alloc -Z unstable-options \
-		    --target=$(srctree)/arch/$(SRCARCH)/rust/target.json
+KBUILD_RUSTCFLAGS := --emit=dep-info,obj,link -Z binary_dep_depinfo=y \
+		     -C panic=abort -C embed-bitcode=n -C lto=n -C rpath=n \
```

Comment:
The target configs define `"panic-strategy": "abort",`, so `-Cpanic=abort` shouldn't be necessary I think.

Reply:
I put it there for completeness, i.e. to document that we always expect to compile with abort. But yeah, it is a noop unless an arch maintainer is doing something funny.